pages/your_tasks/software_documentation.md

```diff
@@ -99,16 +100,18 @@ The `help` command should cover:
 * And ideally, some examples.
 
 Tools like [Click](https://click.palletsprojects.com/en/8.1.x/) for Python can assist you not only in creating your help command but also in building your interface.
-
 A great example of a good CLI is the one included with the [Magic-BLAST](https://ncbi.github.io/magicblast/) bioinformatics tool.
 
+The [OpenAPI Specification][open-ai], previously known as the Swagger Specification, is a specification for a machine-readable API definition language for describing, producing, consuming and visualising web services.

```

Comment:
`[OpenAPI Specification][open-ai]` is this a typo or intended?

Reply:
Apologies - where it the typo @jankapunkt?

Reply:
Ah, got you!